client/src/components/RadioButtons/RadioButtons.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import { RadioButtons } from "./RadioButtons";
+
+export default {
+  title: "Input/RadioButton",
+  component: RadioButtons,
+};
+
+const Template = (args) => <RadioButtons {...args} />;
+
+export const Basic = Template.bind({});
+Basic.args = {
+  options: [
+    {
+      display: "Option 1",
+      id: "1",
+      active: false,
+    },
+    {
+      display: "Option 2",
+      id: "2",
+      active: false,
+    },
+  ],
+};
```

Comment:
To cover all styling cases, need three or four items I think. Maybe have two or three stories for different numbers of options? Editing the `options` value in the controls to add a new item seems fiddly, if even possible (although it does work for seeing what they look like when active).

Another case where I think the call back prop `onChange` should log clicks as storybook actions (however that works, I still haven't read that part of the docs, ha).

Reply:
Still applies